CHANGELOG.md

```diff
@@ -462,6 +463,7 @@ https://textual.textualize.io/blog/2022/11/08/version-040/#version-040
 - New handler system for messages that doesn't require inheritance
 - Improved traceback handling
 
+[0.11.0]: https://github.com/Textualize/textual/compare/v0.10.0...v0.11.0
```

Comment:
v0.10.1 is missing in here, should we add it?

Reply:
Will do